src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/trust/TrustDataDto.java

```diff
@@ -10,16 +10,21 @@
 
 public class TrustDataDto {
 
-    @JsonProperty("trust_id")
+    public static final String TRUST_ID_FIELD = "trust_id";
+    public static final String TRUST_NAME_FIELD = "trust_name";
+    public static final String CREATION_DATE_FIELD = "creation_date";
+    public static final String UNABLE_TO_OBTAIN_ALL_TRUST_INFO = "unable_to_obtain_all_trust_info";
```

Comment:
I would append `_FIELD` to the name of this constant, to be consistent with the others you've created.

Reply:
thanks @mwestacott . I have corrected the field name.